app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -113,6 +118,112 @@ fun ApprovalButton(
     }
 }
 
+@Composable
+        /**
+         * @param dialogIcon Icon to be displayed in the dialog.
+         * @param title The Dialog's title text.
+         * @param content The text content of the dialog. Can be left empty if it is an input dialog
+         * @param onPositiveClick The callback action for clicking the positive approval button.
+         * @param onNegativeClick The callback action for clicking the negative approval button.
+         * @param isTextInputAvailable if set to true, the dialog would include an input text field.
+         * @param textInputValue The text content of the dialog. Can be left empty if it is an input dialog
+         * @param showPositiveButtonDialog Determines whether to display the positive button in the dialog.
+         * @param showNegativeButtonDialog Determines whether to display the negative button in the dialog.
+         */
+fun CustomDialog(
+    dialogIcon: Painter = painterResource(id = R.drawable.greenstand_logo),
+    title: String = "",
+    content: String = "",
+    onPositiveClick: () -> Unit = {},
+    onNegativeClick: () -> Unit = {},
+    isTextInputAvailable: Boolean = false,
+    textInputValue: String = "",
+    onTextInputValueChange: (String) -> Unit = {},
+    showPositiveButtonDialog: Boolean = true,
+    showNegativeButtonDialog: Boolean = false,
```

Comment:
We can use null onPositive/onNegativeClick params to determine if we show the buttons. 
e.g. If we want to positive only button, we only pass that click listener. Since the negative click listener is null, it is turned off.